src/imageuploadcommand.js

```diff
@@ -28,6 +28,7 @@ export default class ImageUploadCommand extends Command {
 	 * @fires execute
 	 * @param {Object} options Options for executed command.
 	 * @param {File} options.file Image file to upload.
+	 * @param {module:engine/model/position~Position} [options.insertPosition] Position of the inserted image.
```

Comment:
Could be both perhaps, but `insertionPosition` is longer and less adequate. If it was an event, then `insertionPosition` would be good because the insertion already took place. So it's already a "position of insertion". Here we want a position to insert the image at. Hence, `insertAt`.

The names of variables in the code can follow the param naming, it's always good. But the variable names are not good now too :P So they should rather be changed, not exposed.